stripevpaypal.md

```diff
@@ -0,0 +1,115 @@
+# Stripe and PayPal Payment Setup on Squarespace  
+
+****Authored by:****  [Ryan Lang](https://github.com/langrh), 1st yr. college    
+
+****Last Updated:**** September 21, 2023
+
+## Overview
+
+Stripe and PayPal are two payment types on Squarespace, each with their own pros and cons. However, both can be used simultaneously on your website. This guide will explain how to set up and use both payment processors, along with a comparison between them.
+
+## How to Set Up
+
+1. On Squarespace, navigate to *Commerce -> Payments*.
+2. Once there, PayPal and Stripe will show up as payment methods.
+3. For both, an account must be linked to use.
+4. To create an account for Stripe, navigate to [Stripe Registration](https://dashboard.stripe.com/register).
+5. To create an account for PayPal, navigate to [PayPal Business Signup](https://www.paypal.com/bizsignup).
+6. Once an account is created, link the account on Squarespace and it will be ready for use.
+
+## Stripe and PayPal Payment Comparison on Squarespace
+
+| Aspect                       | PayPal                                                                                                    | Stripe                                                                                     |
+|------------------------------|-----------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|
+| **Ease of Use**              | Squarespace makes integration simple. Users can use existing or set up a new account through Squarespace. | Seamless integration with just a Stripe account.                                           |
+| **Transaction Fees**         | Varies by user's location.                                                                                | Varies by country.                                                                         |
+| **Processing**               | Allows checkout without sharing financial info on the merchant site. Payment occurs on PayPal’s website.  | Payments take place on the merchant site with Stripe, providing a simpler user experience. |
+| **Accepted Payment Methods** | PayPal Balance, Visa, Mastercard, American Express, Discover, PayPal Credit.                              | Major credit and debit cards, Apple Pay, Google Pay.                                       |

```

Comment:
This means both accept major credit and debit cards, right? If there are similarities between the two, the same language should be used between the two.

Reply:
Updated Accepted Payment Methods and added sources.